zulipterminal/scripts/render_symbols.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+import urwid
+
+from zulipterminal.config import symbols
+
+
+palette = [
+    ('header', 'white, bold', 'light blue'),
+    ('footer', 'black', 'white'),
+]
+
+variables = vars(symbols)
+symbol_dict = {symbol: variables[symbol] for symbol in variables
+               if not symbol.startswith('__') and not symbol.endswith('__')}
+
+symbols_list = [urwid.Columns([urwid.Text(symbol, align='center'),
+                               urwid.Text(symbol_dict[symbol], align='center')
+                               ]) for symbol in symbol_dict]
+symbols_display_box = urwid.Pile(symbols_list)
+
+line_box = urwid.LineBox(symbols_display_box,
+                         title="Render Symbols",
+                         title_attr='header',
+                         tlcorner='┌', tline='─', lline='│',
+                         trcorner='┐', blcorner='└', rline='│', bline='─',
+                         brcorner='┘')
```

Comment:
These all are default parameters of `LineBox`, so no need to specify explicitly?